domain/src/main/java/starter/project/domain/Car.java

```diff
@@ -5,12 +5,18 @@
 import lombok.Data;
 import lombok.NoArgsConstructor;
 
+import javax.persistence.Entity;
+import javax.persistence.Id;
```

Comment:
These two imports are not being used. You can have intellij optimise the imports for this class

Reply:
resolved